yapapi/executor/__init__.py

```diff
@@ -190,9 +203,15 @@ def input_tasks() -> Iterable[Task[D, R]]:
         offers_collected = 0
         proposals_confirmed = 0
 
+        def allocation_for_platform(payment_platform: str) -> rest.payment.Allocation:
+            try:
+                return next(
+                    a for a in self._budget_allocations if a.payment_platform == payment_platform
```

Comment:
May I suggest to use more descriptive name instead of `a` here?

Reply:
Fixed